libs/command_line_handling/src/command_line_handling.cpp

```diff
@@ -1451,22 +1516,6 @@ namespace hpx { namespace util
         if (node_ == std::size_t(-1))
             node_ = 0;
 
-#if defined(HPX_HAVE_NETWORKING)
-#if defined(HPX_HAVE_PARCELPORT_MPI)
-        // getting localities from MPI environment (support mpirun)
-        if (util::mpi_environment::check_mpi_environment(rtcfg_))
-        {
-            mpi_environment::init(&argc, &argv, *this);
-            num_localities_ = static_cast<std::size_t>(mpi_environment::size());
-        }
-#endif
-
-        if (num_localities_ != 1 || node_ != 0 || rtcfg_.enable_networking())
-        {
-            parcelset::parcelhandler::init(&argc, &argv, *this);
-        }
-#endif
-
```

Comment:
Where does this code live now?

Reply:
In `hpx_init.cpp`: https://github.com/STEllAR-GROUP/hpx/blob/c6192fff99f1f71dfc37dea8518bd668c46b02d0/src/hpx_init.cpp#L703-L719. It may not be the best final place, but I'd like to separate setting up configuration options and actually acting on them as much as possible.

Clearly this (or #4206) isn't working right though with tests failing so you might want to hold off looking at this too deeply (I meant to make this a draft pull request).

Reply:
Ok, thanks!